docs/core-concepts/packages.md

```diff
@@ -0,0 +1,170 @@
+- [System.IO.FileSystem](https://www.nuget.org/packages/System.IO.FileSystem) - A set of types for reading and writing to local or networked disk-based storage, including [File](http://dotnet.github.io/api/System.IO.File) and [Directory](http://dotnet.github.io/api/System.IO.Directory).
+- [System.Linq](https://www.nuget.org/packages/System.Linq) - A set types for querying objects, including Enumerable and [ILookup&lt;TKey, TElement&gt;](http://dotnet.github.io/api/System.Linq.ILookup`2);.
+- [System.Reflection](https://www.nuget.org/packages/System.Reflection) - A set of types for loading, inspecting and activating types, including [Assembly](http://dotnet.github.io/api/System.Reflection.Assembly), [TypeInfo](http://dotnet.github.io/api/System.Reflection.TypeInfo) and [MethodInfo](http://dotnet.github.io/api/System.Reflection.MethodInfo).
+
+Packages are referenced in project.json. In the example below, the [System.Runtime](https://www.nuget.org/packages/System.Runtime/) package is referenced. 
+
+```json
+{
+  "dependencies": {
+    "System.Runtime": "4.1.0"
+  },
+  "frameworks": {
+    "netstandard1.5": {}
+  }
+}
+```
+
+In most cases, you will not reference the lower-level .NET Core packages directly since you'll end up with too many packages to manage. Instead, you'll reference a metapackage.
+
+Metapackages
+============
+
+Metapackages are a NuGet package convention for describing a set of packages that are meaningful together. They represent this set of packages by making them dependencies. They can optionally establish a target framework for this set of packages by specifying a framework. 
+
+By referencing a metapackage, you are, in effect, adding a reference to each of its dependent packages as a single gesture. That means that all of the libraries in those packages (refs or libs) are available for IntelliSense (or similar experience) and for publishing (libs only) your app. 
+
+Note: The 'lib' and 'ref' terms refer to folders in NuGet packages. 'ref' folders describe the public API of a package via assembly metadata. 'lib' folders contain the implementation of that public API for a given target framework. 
+
+There are advantages to using metapackages:
+
+- Provides a convenient user experience to reference a large set of fine-grained packages. 
+- Defines a set of packages (including specific versions) that are tested and work well together.
+
+These are the key .NET Core metapackages:
+
+- [Microsoft.NETCore.App](https://www.nuget.org/packages/Microsoft.NETCore.App) - Describes the libraries that are part of the .NET Core distribution (including the libraries that are part of the .NET Core SDK). Establishes the [`netcoreapp` framework](https://github.com/dotnet/core-setup/blob/master/pkg/projects/Microsoft.NETCore.App/Microsoft.NETCore.App.pkgproj). 
```

Comment:
The thing I would want to know about the package is what dependencies it has. And that's already listed on the nuget.org page. Though [the project.json](https://github.com/dotnet/core-setup/blob/master/pkg/projects/Microsoft.NETCore.App/project.json) is where most of it is defined, so maybe that could be of interest?

Or maybe link to [the whole directory](https://github.com/dotnet/core-setup/tree/master/pkg/projects/Microsoft.NETCore.App)?